src/pages/boards/index.vue

```diff
@@ -26,16 +26,14 @@ const boards = ref<Partial<Board>[]>([
   },
 ]);
 
-function createBoard() {
-  console.log("board created");
-}
+// function createBoard() {}
```

Comment:
I think it would be better if we just add
```suggestion
function createBoard() {
 // @todo add create board logic here
}
```